src/vs/workbench/parts/files/electron-browser/views/explorerViewer.ts

```diff
@@ -533,7 +533,11 @@ export class FileDragAndDrop implements ITreeDragAndDrop<ExplorerItem> {
 		return false;
 	}
 
-	getDragURI(element: ExplorerItem): string {
+	getDragURI(element: ExplorerItem): string | null {
+		if (this.explorerService.isEditable(element)) {
+			return null;
```

Comment:
Finally, we just return `null` as a drag URI for editable elements... they should never be dragged.